src/patterns/DetailList/DetailList.tsx

```diff
@@ -0,0 +1,64 @@
+import { ComponentProps, forwardRef } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import { DetailComponent } from '../../components/Detail';
+
+// You'll need to change the ComponentProps<"htmlelementname"> to match the top-level element of your component
+export interface DetailListProps extends ComponentProps<'div'> {
+  /**
+   * Determines whether each Details' label and value are aligned in columns or justified
+   */
+  alignment: 'columns' | 'justified';
```

Comment:
this is a great idea @aahill50.  It might be good to define a `DetailsListAlignment` enum and export it

Reply:
Done